org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterTemperature.java

```diff
@@ -79,7 +92,7 @@ public void attributeUpdated(ZclAttribute attribute) {
                 && attribute.getId() == ZclTemperatureMeasurementCluster.ATTR_MEASUREDVALUE) {
             Integer value = (Integer) attribute.getLastValue();
             if (value != null) {
-                updateChannelState(new DecimalType(BigDecimal.valueOf(value, 2)));
+                updateChannelState(new QuantityType<Temperature>(BigDecimal.valueOf(value, 2), SIUnits.CELSIUS));
```

Comment:
Minor comment: One could omit the type parameter for the `QuantityType` here for brevity, as it can be inferred by the type checker as `SIUnits.CELSIUS` is a `Unit<Temperature>`:

```
updateChannelState(new QuantityType<>(BigDecimal.valueOf(value, 2), SIUnits.CELSIUS));
```

(The same applies to the `QuantityType<Pressure>` in the converter for pressure above.)

This is probably a matter of personal preference, but if you like the shorter version as well, you could change it :)